### PR TITLE
luanaruiz9 tfds cartesian

### PR DESCRIPTION
**Pull request by luanaruiz9**
Included tfds class for the cartesian product dataset. Moved example_generator.py and README.md to the main data folder, so that they can be used for both the PCFG and the cartesian product datasets.

### CODE REVIEW

File: datasets/cartesian/tensorflow-datasets/cartesian.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        homepage='',
+        citation=_CITATION,
+    )
+
+  def _split_generators(self, dl_manager: tfds.download.DownloadManager):
+    """Returns SplitGenerators."""
+    archive_path = dl_manager.manual_dir / 'data'
+    extracted_path = dl_manager.extract(archive_path)
+    return {
+      # original data
```
Comment (santiaontanon-google) [inline datasets/cartesian/tensorflow-datasets/cartesian.py]: nit: Use proper capitalization and punctuation for comments, e.g., here: "Original data.". Same for the others below

File: datasets/pcfg/tensorflow-datasets/pcfg_full_data.py [hunk 2]
```diff
@@ -1,9 +1,14 @@
 """pcfg_full_data dataset."""
 
-from example_generator import example_generator
+import sys
+
 import tensorflow as tf
 import tensorflow_datasets as tfds
 
+sys.path.append('../../datasets')
```
Comment (santiaontanon-google) [inline datasets/pcfg/tensorflow-datasets/pcfg_full_data.py]: I know this is common in Python, but is there a cleaner way? (I don't know myself, so, just asking. If this is the only way, then it's fine)

**Review by santiaontanon-google — APPROVED** [review]
Thanks Luana, looks good, just some tiny nits.

**Review by jtainslie — APPROVED** [review]
Lgtm.

**Pull request merged**